### PR TITLE
Disable PropertyVariableDeclaration if outputLanguage is es6

### LINKED ISSUE #1788: memberVariables output bogus code with outputLanguage=es6
https://google.github.io/traceur-compiler/demo/repl.html#%2F%2F%20Options%3A%20--annotations%20--member-variables%20--output-language%3Des6%20--source-maps%20--type-assertions%20--types%20%0Aclass%20MyType%20%7B%7D%0Aclass%20C%20%7B%0A%20%20foo%3AMyType%3B%0A%7D%0A

for an example --- this is causing issues for "prod" builds of angular with outputLanguage=es6 (which is subsequently re-compiled to output as es5)

screen shot 2015-03-03 at 2 46 01 pm

### PR DESCRIPTION
**Pull request by caitp**
Closes #1788

### CODE REVIEW

**Comment by caitp** [comment]
I'll add tests if anyone can give me an idea where to put them --- I don't see any existing tests for memberVariables support

**Comment by caitp** [comment]
added a basic test which at least demonstrates that the gunk isn't written in es6 mode? it's not a very useful test though since it doesn't show the actual output

File: src/outputgeneration/ParseTreeWriter.js [hunk 1]
```diff
@@ -1137,6 +1139,7 @@ export class ParseTreeWriter extends ParseTreeVisitor {
    * @param {PropertyVariableDeclaration} tree
    */
   visitPropertyVariableDeclaration(tree) {
+    if (this.options_.outputLanguage === 'es6') return;
```
Comment (caitp) [inline src/outputgeneration/ParseTreeWriter.js]: On the other hand, why would you ever want to output this stuff?

**Comment by johnjbarton** [comment]
Tests for feature MemberVariables:
https://github.com/google/traceur-compiler/tree/master/test/feature/MemberVariables

File: src/outputgeneration/ParseTreeWriter.js (hunk 1 above)
Comment (arv) [inline src/outputgeneration/ParseTreeWriter.js]: The writers only responsibility is to serialize the AST back to source code. It should not make this kind of decisions.

**Pull request closed without merging**